op-chain-ops/genesis/setters.go

```diff
@@ -89,3 +89,12 @@ func setupPredeploy(db vm.StateDB, deployResults immutables.DeploymentResults, s
 
 	return nil
 }
+
```

Comment:
Could we add a better comment here? This is the contract that holds the ring buffer of CL roots right? Perhaps the source of truth for that code should be superchain registry?

Reply:
added, it's a very short contract, defined by an EIP, rather than solidity source-code. I think it can just be a constant.